test/WebSites/FormatterWebSite/Models/AdminValidator.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Reflection;
+
+namespace FormatterWebSite
+{
+    public class AdminValidator : ValidationAttribute
+    {
+        protected override ValidationResult IsValid(object value, ValidationContext validationContext)
+        {
+            var objectType = value.GetType();
```

Comment:
Shouldn't you just be able to cast to `Administrator` here instead of doing reflection?